diagrams/src/main.js

```diff
@@ -15,8 +15,8 @@
     return {
         drawDiagramsFromParserInstance: function(parserInstanceToDraw, targetDiv) {
             var topRules = parserInstanceToDraw.getGAstProductions().values()
-            targetDiv.innerHTML = diagrams_builder.buildSyntaxDiagramsText(topRules);
-            diagrams_behavior.initDiagramsBehavior()
+            targetDiv.innerHTML = builder.buildSyntaxDiagramsText(topRules);
+            builder.initDiagramsBehavior()
```

Comment:
fixed in https://github.com/SAP/chevrotain/commit/184ce0d3c417f3f3ac949f40bc569b8cb46d790f

I really must create some tests for the Diagrams, pure Javascript is too forgiving for common mistakes :cry: